### PR TITLE
Add Moto X (ghost)

### PR DESCRIPTION
**Pull request by Parthav46**

### CODE REVIEW

File: manifests/motorola_ghost.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+
```
Comment (jbruechert) [inline manifests/motorola_ghost.xml:6]: You can remove this newline

File: manifests/motorola_ghost.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  <project path="device/motorola/ghost" name="android_device_motorola_ghost" remote="los" revision="cm-13.0"/>
+
+  <project path="device/motorola/msm8960dt-common" name="android_device_motorola_msm8960dt-common" remote="los" revision="cm-13.0"/>
+
+  <project path="vendor/motorola" name="proprietary_vendor_motorola" remote="them"/>
+
```
Comment (jbruechert) [inline manifests/motorola_ghost.xml:11]: same here

File: manifests/motorola_ghost.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+
+  <project path="device/motorola/ghost" name="android_device_motorola_ghost" remote="los" revision="cm-13.0"/>
```
Comment (jbruechert) [inline manifests/motorola_ghost.xml]: Are you sure the cm-13.0 branch is working with Halium's 14.1 tree? There might  be a fork of the repository with 14.1 support.

Comment (Parthav46) [reply manifests/motorola_ghost.xml]: The latest branch on LineageOS GitHub for ghost is of cm-13.0. I am not sure of how to check if the branch is working with Halium's 14.1 tree. Can you provide any assistance for what to check?

Comment (lnjX) [reply manifests/motorola_ghost.xml]: You can just test if your port works / see which problems there are.

**Pull request merged**